benchmark/Benchmark/Compare.hs

```diff
@@ -0,0 +1,45 @@
+module Benchmark.Compare
+       (main
+       ) where
+
+import Data.Text (Text)
+import Gauge.Main (bench, bgroup, defaultMain, whnf)
+import Text.Toml (parseTomlDoc)
+import Text.Toml.Types (Table)
+
+import Benchmark.Htoml (convertHtoml, decodeHtoml)
+import Benchmark.Tomland (codec)
+import Toml.Bi.Code (decode, runTomlCodec)
+import Toml.Parser (parse)
+import Toml.Type (TOML)
+
+import qualified Data.Text.IO as TIO
+
+
+-- | Benchmark.
+main :: IO ()
+main = do
+    (txt, htoml, tomland) <- preload
+    defaultMain
+        [ bgroup "tomland"
```

Comment:
I feel like instead of grouping by library it makes more sense to group by stage. So you can visually compare, for example, now parsing works for different libraries, because in the terminal they all will be near each other.